tests/collection.rs

```diff
@@ -333,9 +327,321 @@ async fn test_get_checksum() {
     assert_eq!(updated_result.name, collection_name);
     assert_eq!(updated_result.is_system, false);
     assert_eq!(updated_result.status, 3);
-    assert_eq!(updated_result.r#type, 2);
+    assert_eq!(updated_result.r#type, CollectionType::Document);
     assert_eq!(updated_result.checksum.is_empty(), false);
 
     let coll = database.drop_collection(collection_name).await;
     assert_eq!(coll.is_err(), false);
 }
+
+#[maybe_async::test(
```

Comment:
In the latest version of `maybe-async`, the `test` API has changed and is able to specify multiple async test macros.

In short, use the following code:

```rust
#[maybe_async::test(
    any(feature = "reqwest_blocking"),
    async(any(feature = "reqwest_async"), tokio::test),
    async(any(feature = "surf_async"), async_std::test)
)]
async fn test_put_load() {
// ...
}
```